docker-compose.yml

```diff
@@ -8,11 +8,18 @@ services:
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=postgres
   web:
-    build: .
-    command: python3 manage.py migrate --noinput && python3 manage.py runserver 0.0.0.0:8000
+    build: ./src/django
```

Comment:
We usually put application source into a sub-directory so that it can be separated from deployment code, utility scripts, etc. There's not a standardized company-wide directory structure, but something like this would be reasonable.